apps/ligo-virgo/src/pages/workspace/docs/Page.tsx

```diff
@@ -105,6 +130,8 @@ const EditorContainer = ({
             const obv = new ResizeObserver(e => {
                 setPageClientWidth(e[0].contentRect.width);
             });
+
+            getEditor(editorRef.current);
```

Comment:
This operation is UNSTABLE and UNSAFE!

Reply:
why, I can't understand it.

Reply:
maybe add?

<img width="404" alt="image" src="https://user-images.githubusercontent.com/110213079/186347568-db914995-8b6e-45ff-8cb4-22049f156f96.png">

Reply:
There's `useCurrentEditors` now

Reply:
yep